### PR TITLE
Feedback for contributor/outline

### PR DESCRIPTION
**Pull request by rrrutledge**
Looping in after-merge feedback from https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/59.  Thanks, @gruetter!

### CODE REVIEW

File: contributor/outline.md [hunk 1]
```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
...
   - Using shared solutions puts you in a position to become a _Contributor_.
     - There will be times when you wish that the shared solution has some new feature.
-    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
+    - Don't _Wait it Out_ or _Work Around_; spend your time and energy making the change directly in the shared project.
     - Briefly enumerate the benefits of contributing.
     This should be taken from the **Benefits** segment.
-    - You may need to spend a bit of time explaining these benefits to your management hierarchy in order to have their support to spend your time contributing.
-    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as long as they get the needed functionality.
-  - Contributing applies to all levels of project managements, not just code.
+    - You may need to spend a time explaining these benefits to your management hierarchy in order for their support for the time you spend contributing.
```
Comment (guywmartin) [inline contributor/outline.md]: Small typo - 'You may need to spend *some* time explaining..

File: contributor/outline.md [hunk 2]
```diff
@@ -67,12 +68,13 @@ Outline for this section inspired by the [Trusted Committer introduction video].
     - Feedback from the host team is not any kind of personal criticism to you.
     Treat and value it as data to help your contribution become all the better.
     - If the host team decides not to accept a contribution, don't take it personally.
+    Work with them to understand their reasoning.
     Remember and respect that the host team has ultimate decision over the project.
     They're the ones that are signing up to maintain it, after all!
     Try to collaborate on an aspect of the contribution that you all agree on,
     and look for another way to fill the rest of your needs.
   
-## Mechanics of Contributing
```
Comment (guywmartin) [inline contributor/outline.md]: Why did we change this?  I think 'Mechanics of Contributing' actually works better as a title...

File: contributor/outline.md (hunk 1 above)
Comment (rrrutledge) [reply contributor/outline.md]: Good catch.

File: contributor/outline.md (hunk 2 above)
Comment (rrrutledge) [reply contributor/outline.md]: This was feedback from @gruetter on [the previous PR](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/59).  Although on second thought I think I agree with you.

File: contributor/outline.md [hunk 3]
```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
 
 ## Becoming an InnerSource Contributor
 
-  - A contributing mindset starts with a shared use mindset.
-    - When you have technical needs that are outside of the domain of your project,
+  - A contributing mindset starts with a sharing mindset.
+    - When you have technical needs that are orthoganal of the domain of your project,
```
Comment (lenucksi) [inline contributor/outline.md]: I'm pretty sure you have changes that are **orthogonal** not **orthoganal**, also they likely are orthogonal **to** not **of**? 
```suggestion
    - When you have technical needs that are orthogonal to the domain of your project,
```

File: contributor/outline.md [hunk 4]
```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
...
     - There will be times when you wish that the shared solution has some new feature.
-    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
+    - Don't _Wait it Out_ or _Work Around_; spend your time and energy making the change directly in the shared project.
     - Briefly enumerate the benefits of contributing.
     This should be taken from the **Benefits** segment.
-    - You may need to spend a bit of time explaining these benefits to your management hierarchy in order to have their support to spend your time contributing.
-    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as long as they get the needed functionality.
-  - Contributing applies to all levels of project managements, not just code.
+    - You may need to spend some time explaining these benefits to your management hierarchy in order for their support for the time you spend contributing.
+    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as it's in a well-run project that gives them the functionality they need.
```
Comment (lenucksi) [inline contributor/outline.md]: Nice! :+1:

File: contributor/outline.md [hunk 5]
```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
...
+  - The scope of contributions goes beyond "just code".
     - Opening issues.
     - Improving documention.
     - Fixing builds.
+    - A test case that shows a bug in the existing code base.
```
Comment (lenucksi) [inline contributor/outline.md]: :D Extended this thing here - we all may have seen test cases that either are flaky or incorrect, i.e. don't test what they should.
```suggestion
    - Creating or fixing test cases that show a bug in the existing code base.
```

File: contributor/outline.md [hunk 6]
```diff
@@ -67,12 +68,13 @@ Outline for this section inspired by the [Trusted Committer introduction video].
     - Feedback from the host team is not any kind of personal criticism to you.
     Treat and value it as data to help your contribution become all the better.
     - If the host team decides not to accept a contribution, don't take it personally.
+    Work with them to understand their reasoning.
```
Comment (lenucksi) [inline contributor/outline.md]: Great, shouldn't this be indented to the level of line 70 though?
```suggestion
      - Work with them to understand their reasoning.
```

File: contributor/outline.md (hunk 2 above)
Comment (lenucksi) [reply contributor/outline.md]: I would agree with @rrrutledge and @guywmartin. Also, I think that mechanics would describe the happy path / clean case of contributions, while the _Gotchas_ surely exist and to me would describe corner cases and other non-standard things.

File: contributor/outline.md (hunk 6 above)
Comment (rrrutledge) [reply contributor/outline.md]: It's meant to just be another sentence on the previous bullet.  You can see that, when rendered, it all forms a paragraph together.

**Comment by rrrutledge** [comment]
Addressed all comments.

File: contributor/outline.md (hunk 6 above)
Comment (lenucksi) [reply contributor/outline.md]: True, my bad, my mental Markdown Renderer apparently wasn't up to speed. ;) 
Closing this one then.

**Comment by rrrutledge** [comment]
Thanks!  Sounds good.

**Pull request merged**